src/static/LecturesDetailPage/classTime.js

```diff
@@ -0,0 +1,25 @@
+export const classTime = [
```

Comment:
상수는 underScore 안하기로 했었죠...!! 
저도 3~4월에 했던 작업들 이렇게 해놔서 반성하고 있습니다 ㅜ

```suggestion
export const CLASS_TIME = [
```